app/assets/javascripts/app/_fpa_loaded_registrations.js

```diff
@@ -0,0 +1,77 @@
+_fpa.loaded.registrations = () => {
+
+    /**
+     see also, config/initializers/app_settings.rb GdprCountryCodes
+
+     https://www.gdpradvisor.co.uk/gdpr-countries
+     Austria AT
+     Belgium BE
+     Bulgaria BG
+     Croatia HR
+     Cyprus CY
+     Czech Republic CZ
+     Denmark DK
+     Estonia EE
+     Finland FI
+     France FR
+     Germany DE
+     Greece GR
+     Hungary HU
+     Ireland IE
+     Italy IT
+     Latvia LV
+     Lithuania LT
+     Luxembourg LU
+     Malta MT
+     Netherlands NL
+     Poland PL
+     Portugal PT
+     Romania RO
+     Slovakia SK
+     Slovenia SI
+     Spain ES
+     Sweden SE
+     United Kingdom GR
+     */
+
+    const GDPR_COUNTRY_CODES = ['AT', 'BE', 'BG', 'HR', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HU',
```

Comment:
I'd like to change this so that the GDPR country codes are passed from the app_settings.rb value. The file app/views/layouts/_setup_app.html.erb already does things like this. Specify something like `_fpa.gdpr_county_codes = Settings::...` like we have done on line 24 of that file. Then refer to that property in here.

Reply:
My latest commits addressed this requested change.